README.md

```diff
@@ -223,7 +223,7 @@ This is implemented in a way that is reasonably agnostic to the exact GPU-Tensor
 1. If you are using a virtual/conda environment, make sure to use the same environment throughout the rest of the build
 1. Build using `./build.sh` or `./build.sh --headless` for headless systems (i.e. without an attached display) or if you need multi-gpu support
 1. Test the build as described above (except that the interactive viewer is at `build/utils/viewer/viewer`).
-1. For use with [Habitat-API](https://github.com/facebookresearch/habitat-api) and your own python code, add habitat-sim to your `PYTHONPATH` (not necessary if you used `python setup.py install` instead of `build.sh`). For example modify your `.bashrc` (or `.bash_profile` in Mac osx) file by adding the line:
+1. For use with [Habitat-API](https://github.com/facebookresearch/habitat-api) and your own python code, add habitat-sim to your `PYTHONPATH` (not necessary if you used `python setup.py install` instead of `build.sh`). For example modify your `.bashrc` (or `.bash_profile` in MacOS) file by adding the line:
```

Comment:
While we're at it -- they kinda insist for it to be `macOS`, uncapitalized, to be consistent with `iOS`, `tvOS`, `iPhone` etc. :)